.env.example

```
@@ -14,6 +14,23 @@ GEMINI_API_KEY=your_gemini_api_key_here
+# Step 1: Enable 2-Step Verification on your Google account (if not already):
+#         https://myaccount.google.com/security
+#
+# Step 2: Create an App Password:
+#         https://myaccount.google.com/apppasswords
+#         → Select app: Mail  → Select device: Other → name it "career-ops"
+#         → Copy the 16-character password (with or without spaces)
+#
+# Step 3: Fill in below and save as .env (not .env.example)
+#
+GMAIL_USER=your@gmail.com
+GMAIL_APP_PASSWORD=xxxx xxxx xxxx xxxx
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GMAIL_APP_PASSWORD key should go before the GMAIL_USER key

(UnorderedKey)